src/states/bootstrapping_peer.rs

```diff
@@ -291,8 +289,16 @@ impl Base for BootstrappingPeer {
         p2p_node: P2pNode,
         _: &mut dyn EventBox,
     ) -> Result<Transition, RoutingError> {
-        // TODO: ignore messages from peers we don't send `BootstrapRequest` to and disconnect
-        // from them.
+        // Ignore messages from peers we didn't send `BootstrapRequest` to.
```

Comment:
Should we disconnect earlier, i.e when we get a connected_to?
Is there a way to ensure that we won't completely receive the message sent to us so the sender know it failed, otherwise, we are not really suppose to drop it as it could be a signature accumulation or other things, but maybe it is ok for now and we can look at it later.

Reply:
> Should we disconnect earlier, i.e when we get a connected_to?

Yeah, that would possibly be a bit more robust.

> Is there a way to ensure that we won't completely receive the message sent to us so the sender know it failed, 

Hm, I'm not sure. Ideally the nodes in the source section would not try to send any messages to the relocated node once they know that it got relocated (so once `Relocate` is accumulated, not just once `SectionInfo` is). Not sure we can do that.

> we are not really suppose to drop it as it could be a signature accumulation

We are not member of the section anymore, so what else are we supposed to do?

Reply:
> m, I'm not sure. Ideally the nodes in the source section would not try to send any messages to the relocated node once they know that it got relocated 

=> Yes, it could also be from other section relaying message.

> We are not member of the section anymore, so what else are we supposed to do?

=> Tricky, not sure, for RoutingMessage, easier, we could send them to all the node we knew as elder, for DirectMessage, that seem difficult to do anything, Maybe we still process them in some way. For example, we may have been sig accumulating and waiting for additional signatures, then we get a new one after we Relocated (because other node was late), then we probably should finish accumulation and send that message, maybe could send it to all elders of section we were at?
=> It may also become less of an issue with node ageing if we don't imediatly relocate elder, i.e we leave them to cool down for a while as non participating node that complete whatever they were doing, and after enough time that we are unlikely to get message we need to process then we relocate.

Reply:
OK, so what about addressing this only when node ageing is in place? Or do you think this causes issue right now?

Reply:
"but maybe it is ok for now and we can look at it later."
I think if we don't get error we can leave it. and yes addressing it with node ageing is probably easier. (But if we get error from message it could also come from there).